src/scripting/game_lua_kernel.cpp

```diff
@@ -2858,6 +2858,9 @@ int game_lua_kernel::intf_set_side_id(lua_State *L)
 	if(flag.empty() && color.empty()) {
 		return 0;
 	}
+	if(team_i < 0 || team_i >= teams().size()) {
+		return luaL_error(L, "set_side_id: side number %d out of range", team_i);
```

Comment:
This produces a type mismatch warning, which causes at least the Xcode build to fail.